app/src/main/java/info/nightscout/androidaps/workflow/PrepareBasalDataWorker.kt

```diff
@@ -116,27 +118,27 @@ class PrepareBasalDataWorker(
         // create series
         data.overviewData.baseBasalGraphSeries = LineGraphSeries(Array(baseBasalArray.size) { i -> baseBasalArray[i] }).also {
             it.isDrawBackground = true
-            it.backgroundColor = rh.gc(R.color.basebasal)
+            it.backgroundColor = ContextCompat.getColor(ctx, R.color.basebasal)
```

Comment:
why not `rh.gac()` here?

Reply:
After 1 day of work i found the solution.
i had to fix gac function. Now it works also her in graph data.

Reply:
> why not `rh.gac()` here?
@MilosKozak : fixed problem. ready for merge